### ISSUE TITLE
Error: The partial swagger/operation could not be found

### ISSUE THREAD

**Issue by joeyfigaro** [anvilco/spectaql#8]
Using a simple config.yml and running `npx spectaql -d ./spectaql.yml` / `yarn dlx spectaql -d ./spectaql.yml` results in the following error: 

```sh
Running "compile-handlebars:compile" (compile-handlebars) task
Task error: [Error: The partial swagger/operation could not be found
  at Object.invokePartial (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:332:11)
  at Object.invokePartialWrapper [as invokePartial] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:84:39)
  at eval (<anonymous>:10:31)
  at prog (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at execIteration (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
  at /Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:89:13
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:91:11)
  at Object.wrapper (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at Object.eval (<anonymous>:13:49)
  at main (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at Object.ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at Object.ret [as swagger/path] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at Object.invokePartialWrapper [as invokePartial] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:88:46)
  at eval (<anonymous>:10:31)
  at prog (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at execIteration (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:51:19)
  at /Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:83:15
  at Array.forEach (<anonymous>:null:null)
  at /Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:78:32
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/each.js:91:11)
  at Object.wrapper (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at eval (<anonymous>:11:49)
  at Object.prog [as inverse] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:27:22)
  at Object.wrapper (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at eval (<anonymous>:10:50)
  at Object.prog [as fn] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:29:22)
  at Object.wrapper (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at Object.eval (<anonymous>:11:47)
  at main (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at Object.ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at Object.ret [as swagger/paths] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at Object.invokePartialWrapper [as invokePartial] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:88:46)
  at eval (<anonymous>:10:31)
  at Object.prog [as inverse] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:268:12)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/helpers/if.js:27:22)
  at Object.wrapper (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/internal/wrapHelper.js:15:19)
  at Object.eval (<anonymous>:19:47)
  at main (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at Object.ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at Object.ret [as layout/content] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at Object.invokePartialWrapper [as invokePartial] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:88:46)
  at Object.eval (<anonymous>:14:28)
  at main (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at Object.ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at Object.ret [as layout/page] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at Object.invokePartialWrapper [as invokePartial] (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:88:46)
  at Object.eval (<anonymous>:13:28)
  at main (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:208:32)
  at ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/runtime.js:212:12)
  at ret (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/handlebars/dist/cjs/handlebars/compiler/compiler.js:519:21)
  at compile (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt-compile-handlebars/tasks/compile-handlebars.js:243:15)
  at /Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt-compile-handlebars/tasks/compile-handlebars.js:277:13
  at Array.forEach (<anonymous>:null:null)
  at /Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt-compile-handlebars/tasks/compile-handlebars.js:276:20
  at Array.forEach (<anonymous>:null:null)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt-compile-handlebars/tasks/compile-handlebars.js:264:11)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/grunt/task.js:252:15)
  at Object.thisTask.fn (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/grunt/task.js:70:16)
  at Object.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:294:30)
  at Task.runTaskFn (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:244:24)
  at Task.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:293:12)
  at Task.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:223:9)
  at Task.runTaskFn (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:247:9)
  at Task.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:293:12)
  at Task.<anonymous> (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:223:9)
  at Timeout._onTimeout (/Users/joeyfigaro/.npm/_npx/96232/lib/node_modules/spectaql/node_modules/grunt/lib/util/task.js:234:33)
  at listOnTimeout (internal/timers.js:531:17)
  at processTimers (internal/timers.js:475:7)
] {
  description: undefined,
  fileName: undefined,
  lineNumber: undefined,
  endLineNumber: undefined,
  message: 'The partial swagger/operation could not be found',
  name: 'Error',
  number: undefined
}
```

**Comment by newhouse** [comment]
Hi @joeyfigaro - very sorry about that. It's a code-path that hasn't been flexed in a while.

Should be fixed now from:
https://github.com/anvilco/spectaql/pull/9
https://github.com/anvilco/spectaql/pull/10

And has been published to `npmjs` version `0.0.7` since you look like you're using the global install...so you'll need to `update`/`updgrade`.

Thanks for using SpectaQL!

**Comment by joeyfigaro** [comment]
@newhouse No worries, mate. Thanks! I'll check it out.
